agent/agentexec/cli_unix_test.go

```diff
@@ -0,0 +1,178 @@
+//go:build linux
```

Comment:
This file should be `cli_linux_test.go` to match the build tags. Unix includes darwin